src/EnergyPlus/DualDuct.cc

```diff
@@ -1042,23 +918,27 @@ namespace DualDuct {
             // The first time through set the mass flow rate to the Max
             // Take care of the flow rates first. For Const Vol and VAV.
             if (this->DamperType == DualDuctDamper::ConstantVolume || this->DamperType == DualDuctDamper::VariableVolume) {
-                if ((state.dataLoopNodes->Node(HotInNode).MassFlowRate > 0.0) && (GetCurrentScheduleValue(state, this->SchedPtr) > 0.0)) {
+                if ((state.dataLoopNodes->Node(HotInNode).MassFlowRate > 0.0) &&
+                    (ScheduleManager::GetCurrentScheduleValue(state, this->SchedPtr) > 0.0)) {
```

Comment:
How many times in this code block are we retrieving the same exact schedule value?